src/tools/compiletest/tidy-rustdoc.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Won't this an issue to have a bash script on windows? Wouldn't a python or a rust one be better?

Reply:
I guess so, but doing this IO in python is super annoying. I'll rewrite it after everything else is working.

Reply:
I moved it to rust instead.